client/components/Editor/views/notePopover.scss

```diff
@@ -9,3 +9,7 @@
 		margin-bottom: 0;
 	}
 }
+
+[data-rtl='true'] {
```

Comment:
⚠️ This should be nested inside `.note-popover-component` so that it doesn't affect everything else in the webapp. Note that we _do_ put the `data-rtl` selector at the top level of `textAlign.scss` because that file is actually `@imported` into a selector in `base.scss`!